application-tour-ui/src/main/resources/TourCode/TourTranslations.xml

```diff
@@ -79,9 +80,9 @@ tour.steps=Steps
 tour.steps.new_step=New step
 tour.steps.add_step=Add step
 tour.steps.edit_step=Edit step
-tour.steps.delete_confirm=Are you sure to delete the step
-tour.steps.delete_done=Delete step done
-tour.steps.deleting=Deleting step
+tour.steps.delete_confirm=Are you sure to delete the step ?
+tour.steps.delete_done=Delete step done !
```

Comment:
Nope. In English, there is no space before ? and !
It's not the same rules than in French, and probably in Romanian neither :)